src/open_inwoner/js/components/form/FileInput.js

```diff
@@ -261,53 +275,57 @@ export class FileInput extends Component {
    * @return {string}
    */
   renderFileHTML(file) {
-    // renderFileHTML is a separate function, where the context of 'this' changes when it is called
     const { name, size, type } = file
     const ext = name.split('.').pop().toUpperCase()
     const sizeMB = (size / (1024 * 1024)).toFixed(2)
     const labelDelete = this.getFilesList().dataset.labelDelete || 'Delete'
     const getFormNonFieldError = this.getFormNonFieldError()
     const formSubmitButton = this.getFormSubmitButton()
 
-    // Only show errors notification if data-max-file-size is exceeded + add error class to file-list
     const maxMegabytes = this.getLimit()
+    const uploadFileTypes = this.getUploadTypes().toUpperCase()
+
+    const sizeError = sizeMB > maxMegabytes
+    const typeError = !uploadFileTypes.includes(ext)
```

Comment:
Not sure if this is a big problem, but because `uploadFileTypes` is a string and not a list with extensions, a file with a space as extension is also allowed (because ` ` occurs in the string):

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/a41f8051-31ac-4b5f-a8e4-dddf46db9760)

Similarly something like "file.pdf, doc" would also be allowed.

I'd turn it into a list to make sure this is not possible:
```suggestion
    const typeError = !uploadFileTypes.split(", ").includes(ext)
```

Reply:
Odd: in all my browsers my front-end is actually already displaying these odd extensions as faulty and returns an error, but I will add this array method anyway because it certainly will not do harm,